packages/toast/toast-container.js

```diff
@@ -14,7 +16,7 @@ import { repeat } from 'lit/directives/repeat.js';
  * @property   {Boolean}                              [canClose]  Whether the toast can be dismissed. Defaults to false. WARNING! For accessibility reasons, toasts should not be interactive and canclose should always be false. If the toast absolutely must be dismissble, set this to true.
  */
 
-export class WarpToastContainer extends WarpElement {
+export class WarpToastContainer extends kebabCaseAttributes(WarpElement) {
```

Comment:
`kebabCaseAttributes` wrapper is not needed here either.

Reply:
Same with SidebarElement and UnstyledHeading components.

Reply:
Updated in this commit: [9ddc7df](https://github.com/warp-ds/elements/pull/174/commits/9ddc7df079fcb7f4c661d5b5b6c0f8150dac7e89)